### PR TITLE
Fix audit2allow test on 15-SP3

### PR DESCRIPTION
**Pull request by paolostivanin**
On 15-SP3, we need to use `$original_audit` instead of the `$audit_log_short`. 

Verification runs:
- 15-SP3:
  - aarch64: https://openqa.suse.de/tests/8796815
  - s390x: https://openqa.suse.de/tests/8796814
  - x86_64: https://openqa.suse.de/tests/8796659
- 15-SP2:
  - aarch64: https://openqa.suse.de/tests/8796816
  - s390x: https://openqa.suse.de/tests/8796817
  - x86_64: https://openqa.suse.de/tests/8796818

### CODE REVIEW

**Comment by rfan1** [comment]
@paolostivanin LGTM
But I have one quick question here, do you have any bug/poo ticket filed? why we have different logic to check the result for SLES15SP3 only?

File: tests/security/selinux/audit2allow.pm [hunk 1]
```diff
@@ -45,8 +46,9 @@ sub run {
     upload_logs($audit_log_short);
 
     # create an SELinux module, make this policy package active, check the new added module
+    $cmd = "cat " . (is_sle('=15-SP3') ? "$original_audit" : "$audit_log_short") . "| audit2allow -M $test_module";
```
Comment (lilyeyes) [inline tests/security/selinux/audit2allow.pm:49]: Add one `" "` (blank) before `"|"` is better.

**Review by lilyeyes — APPROVED** [review]
LGTM.

**Comment by lilyeyes** [comment]
NOTE: https://openqa.suse.de/tests/8796814 failed, maybe can be fixed by rerun.

File: tests/security/selinux/audit2allow.pm (hunk 1 above)
Comment (Amrysliu) [inline tests/security/selinux/audit2allow.pm:49]: How about set a variable for audit log, e.g. `$audit_log = is_sle('=15-SP3') ? "$original_audit" : "$audit_log_short"`, then use this variable in following code.

Comment (paolostivanin) [reply tests/security/selinux/audit2allow.pm:49]: good point :) I'll do this

Comment (paolostivanin) [reply tests/security/selinux/audit2allow.pm:49]: good point, I'll do this :+1:

**Comment by paolostivanin** [comment]
@rfan1 no, I didn't open any bug report. I don't think this is worth a bug, because the "short" audit log is taken (first 500 lines) from the original log. It seems that on openqa the short version doesn't contain the needed messages, but full one does.

**Comment by rfan1** [comment]
> @rfan1 no, I didn't open any bug report. I don't think this is worth a bug, because the "short" audit log is taken (first 500 lines) from the original log. It seems that on openqa the short version doesn't contain the needed messages, but full one does.

Got, that makes sense, then I would suggest you adding some comments there, but not a big deal.

**Review by rfan1 — APPROVED** [review]

**Comment by paolostivanin** [comment]
new verification runs:
- 15-SP3:
  - x86_64: https://openqa.suse.de/tests/8806804
  - aarch64: https://openqa.suse.de/tests/8806805
- 15-SP2:
  - x86_64: https://openqa.suse.de/tests/8806807
  - aarch64: https://openqa.suse.de/tests/8806806

**Pull request merged**